lib/search.js

```diff
@@ -12,7 +12,9 @@ function Search() {
 }
 
 Search.prototype.query = function(q) {
-  return this.index.search(q)
+	return q === '~'
```

Comment:
Not entirely sure what you mean by a wildcard, an "*" (asterisk) ?

If so i can give you another commit with that, or whatever you would like.